frontend/src/metabase/admin/upsells/components/Upsells.styled.tsx

```diff
@@ -55,9 +56,24 @@ export const UpsellCTALink = styled(ExternalLink)`
   }
 `;
 
-export const UpsellCardComponent = styled.div`
-  max-width: 200px;
-  box-sizing: content-box;
+interface Variants {
+  fullWidth?: boolean;
+  maxWidth?: number;
+}
+export const UpsellCardComponent = styled.div<Variants>`
+  ${({ fullWidth, maxWidth }) =>
+    fullWidth
+      ? css`
+          width: 100%;
+        `
+      : maxWidth
```

Comment:
I think it makes sense to accept `maxWidth` if we don't want some title to wrap.

![Screenshot 2024-07-11 at 8 10 09 PM](https://github.com/metabase/metabase/assets/1937582/2b38f72d-f4c1-4fba-b4f6-9a2406a80164)
![Screenshot 2024-07-11 at 8 10 11 PM](https://github.com/metabase/metabase/assets/1937582/1b83df2a-222c-4488-b01f-cfd41782a80a)